mautic/mautic
# TPROD-326 - ReportBundle - Migrate PHP Templates to Twig Templates

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. 



- `/s/reports`
  - same functionality that exists on 5.x
- `/s/reports/new` and `/s/reports/edit/{id}`
  - same functionality that exists on 5.x
- `/s/reports/view/{id}`
  - same functionality that exists on 5.x
- `/s/reports/view/{id}/export/html`
  - same functionality that exists on 5.x

**Known Issues**
- https://github.com/mautic/mautic/issues?q=is%3Aissue+is%3Aopen+report+label%3Areports
- https://github.com/mautic/mautic/issues/11977
